charts/idam-testing-support-api/values.preview.template.yaml

```diff
@@ -13,4 +13,36 @@ java:
     RD_USERPROFILE_API_URL: http://rd-user-profile-api-aat.service.core-compute-aat.internal
     RD_CASEWORKER_API_URL: http://rd-caseworker-ref-api-aat.service.core-compute-aat.internal
     IDAM_S2SAUTH_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
-
+  keyVaults:
```

Comment:
I think the 'values.preview.template.yaml' is only used for the PR pods (not the preview deployment). I was expecting you'd be changing the existing attributes in 'values.yaml' to be applied everywhere as a new chart.

Reply:
Reverted these changes back in 'values.preview.template.yaml' and updated new secret values in values.yml